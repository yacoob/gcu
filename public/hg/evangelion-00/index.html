<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <!--[if lt IE 9]>
  <script src='https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js'></script>
  <script src='https://oss.maxcdn.com/respond/1.4.2/respond.min.js'></script>
  <![endif]-->
  <link href='/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120602_011829.jpg' rel='image_src'/>
  <meta property='og:image' content='https://syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120602_011829.jpg'/>
  <meta name='description' content='tactical grace presents: Evangelion 00'/>
  <link href='http://gcu.tactical-grace.net/hg/evangelion-00/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>HG: Evangelion 00 | tactical grace</title>
  <meta content='HG: Evangelion 00 | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/hg/evangelion-00/'>HG: Evangelion 00</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a href='/hg/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a href='/hg/ahead-smultron/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
    <a href='/hg/exia/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-right'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2012-10-13'>2012-10-13</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120602_014515.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120602_014515.jpg' title='Hiyyaaah!' alt='Hiyyaaah!' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120602_012251.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120602_012251.jpg' title='&amp;quot;!&amp;quot;' alt='&amp;quot;!&amp;quot;' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120602_011829.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120602_011829.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120602_011758.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120602_011758.jpg' title='Can you see the ugliness of the sole? Yup, all hand painted by yours truly.' alt='Can you see the ugliness of the sole? Yup, all hand painted by yours truly.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120602_000055.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120602_000055.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120521_013159.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120521_013159.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120521_013121.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120521_013121.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120519_225638.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120519_225638.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120519_224603.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120519_224603.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120519_214920.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120519_214920.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120519_204110.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120519_204110.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120519_174741.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120519_174741.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120519_000238.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120519_000238.jpg' title='That codpiece was the only place where I used glue, I think.' alt='That codpiece was the only place where I used glue, I think.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120519_000024.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120519_000024.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120518_235753.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120518_235753.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120517_232025.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120517_232025.jpg' title='Temples are now painted by hand, you can tell by wobbliness of the edges.' alt='Temples are now painted by hand, you can tell by wobbliness of the edges.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120517_212056.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120517_212056.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120516_233837.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120516_233837.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120516_222809.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120516_222809.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120516_220804.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120516_220804.jpg' title='Head with stickers.' alt='Head with stickers.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120508_233552.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120508_233552.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/c/2012-10-13/s1920/IMG_20120508_233710.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2012-10-13/s400-p/IMG_20120508_233710.jpg' title='Nape with plug system.' alt='Nape with plug system.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
</div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='http://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link type='text/css' rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/lightgallery/1.3.6/css/lightgallery.min.css' />
<!-- include lightbox js/css here -->
<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js'></script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/lightgallery/1.3.6/js/lightgallery.min.js'></script>
<script src='/js/lg-fullscreen.min.js'></script>
<script src='/js/lg-thumbnail.min.js'></script>
<script src='/js/lg-autoplay.min.js'></script>
<script src='/js/lg-zoom.min.js'></script>
<script src='/js/lg-video.min.js'></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
$(document).ready(gcu.postPageHandler);
</script>
</body></html>