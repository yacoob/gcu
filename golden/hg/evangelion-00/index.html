<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <link href='https://cdn.jsdelivr.net/npm/bootstrap3@3/dist/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='/photos/full/gcu-00029.jpg' rel='image_src'/>
  <meta property='og:image' content='https://gcu.tactical-grace.net/photos/full/gcu-00029.jpg'/>
  <meta name='twitter:card' content='summary_large_image'/>
  <meta name='twitter:image' content='https://gcu.tactical-grace.net/photos/full/gcu-00029.jpg'/>
  <meta name='description' content='tactical grace presents: Evangelion 00'/>
  <meta name='twitter:title' content='tactical grace presents: Evangelion 00'/>
  <link href='https://gcu.tactical-grace.net/hg/evangelion-00/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>HG: Evangelion 00 | tactical grace</title>
  <meta content='HG: Evangelion 00 | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/hg/evangelion-00/'>HG: Evangelion 00</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a data-toggle='tooltip' title='main page' href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a data-toggle='tooltip' title='contact the author' href='https://goo.gl/forms/iJEsfLjBy1FE86aQ2'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-envelope'></span></button></a>
    <a data-toggle='tooltip' title='grade page' href='/hg/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a data-toggle='tooltip' title='previous kit' href='/hg/ahead-smultron/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
    <a data-toggle='tooltip' title='next kit' href='/hg/exia/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-right'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2012-10-13'>2012-10-13</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00031.jpg' data-sub-html='Hiyyaaah!'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00031.jpg' alt='Hiyyaaah!' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00030.jpg' data-sub-html='&amp;quot;!&amp;quot;'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00030.jpg' alt='&amp;quot;!&amp;quot;' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00029.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00029.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00028.jpg' data-sub-html='Can you see the ugliness of the sole? Yup, all hand painted by yours truly.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00028.jpg' alt='Can you see the ugliness of the sole? Yup, all hand painted by yours truly.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00027.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00027.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00026.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00026.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00025.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00025.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00024.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00024.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00023.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00023.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00022.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00022.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00021.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00021.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00020.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00020.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00019.jpg' data-sub-html='That codpiece was the only place where I used glue, I think.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00019.jpg' alt='That codpiece was the only place where I used glue, I think.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00018.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00018.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00017.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00017.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00016.jpg' data-sub-html='Temples are now painted by hand, you can tell by wobbliness of the edges.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00016.jpg' alt='Temples are now painted by hand, you can tell by wobbliness of the edges.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00015.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00015.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00014.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00014.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00013.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00013.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00012.jpg' data-sub-html='Head with stickers.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00012.jpg' alt='Head with stickers.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00010.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00010.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00011.jpg' data-sub-html='Nape with plug system.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00011.jpg' alt='Nape with plug system.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
</div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='https://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/css/lightgallery.min.css" />
<script src="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/js/lightgallery.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-thumbnail.js@1/dist/lg-thumbnail.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-zoom.js@1/dist/lg-zoom.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-video.js@1/dist/lg-video.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-fullscreen.js@1/dist/lg-fullscreen.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-autoplay.js@1/dist/lg-autoplay.min.js"></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
document.addEventListener('DOMContentLoaded', gcu.postPageHandler);
</script>
</body></html>